### ISSUE TITLE
Reproducing SiamMOT

### ISSUE THREAD

**Issue by Leo63963** [Tanmay-Garg01/siam-mot-aot#1]
Hi @Tanmay-Garg01 
Have you reproduced the results of SiamMOT on MOT17 by training yourself? 
Thanks.

**Comment by Tanmay-Garg01** [comment]
Yes

**Comment by Leo63963** [comment]
Hi @Tanmay-Garg01 
Thanks for your reply.
Could you please tell me the parameters for reproducing the results of SiamMOT? 
I follow the official parameters settings at [https://github.com/amazon-research/siam-mot/blob/main/configs/dla/DLA_34_FPN_EMM_MOT17.yaml](url), change the dataset to:  `TRAIN: ( "MOT17",)`, and reduce the learning rate and batch size accordingly since my GPUs are limited. The training takes 30 hours.
The results on the training set are pretty good, with MOTA = 65.4, IDF1 = 63.6, which is a little bit better than the model provided by the author, which can be found at [https://github.com/amazon-research/siam-mot/blob/main/readme/model_zoo.md](url).
However, when testing on MOT17 test sequence, and submit the results to MOTChallenge Benchmark, the evaluated results are much worse than the SiamMOT, which can be found on the corresponding paper (MOTA = 65.9, IDF1=63.3).
Could you tell me the parameters that you used for reproducing the SiamMOT on MOT17?
Thanks.
